src/ceph-disk/ceph_disk/main.py

```diff
@@ -1,12 +1,10 @@
 #!/usr/bin/env python
 #
-# Copyright (C) 2015, 2016 Red Hat <contact@redhat.com>
+# Copyright (C) 2015-2016 Red Hat <contact@redhat.com> Authors
```

Comment:
Does Red Hat recommend that the copyright statement look like this? (The addition of the word "Authors" seems to render the copyright statement ambiguous. To whom does the copyright belong - Red Hat, or its Authors?)